[_]: feat/removed-limited-views-feature

vicblab opened the pull request:
Now the default and only max views of a shared link is infinite.

aboredvaro committed:
Added generic list component

Code needs to be reviewed to make shared links work

aboredvaro committed:
Added empty state and dropdown menu

aboredvaro committed:
Added dialog

foxkdev committed:
share link service

foxkdev committed:
Merge remote-tracking branch 'origin/shared-links-and-trash' into shared-links-and-trash

foxkdev committed:
get share links

aboredvaro committed:
Updated generic list and share dialog

aboredvaro committed:
Added Trash

aboredvaro committed:
Added multiselection and double click support

aboredvaro committed:
WIP

foxkdev committed:
WIP

foxkdev committed:
sdk 1.0.1

aboredvaro committed:
WIP

foxkdev committed:
update share link service

aboredvaro committed:
Display share links and copy link

foxkdev committed:
added deleteShareLink sin share service

foxkdev committed:
Merge remote-tracking branch 'origin/shared-links-and-trash' into shared-links-and-trash

aboredvaro committed:
Fixed scroll issue

aboredvaro committed:
Fixed keyboard control issue

aboredvaro committed:
Added update link

aboredvaro committed:
updated sdk version

aboredvaro committed:
Added infinite scroll pagination and update link

aboredvaro committed:
Fixed z-index issue

AlexMenor committed:
deleteLink in SharedLinksView list

AlexMenor committed:
handle deleteSelectedItems

AlexMenor committed:
refactor generic list component

AlexMenor committed:
make non orderable columns seem not clickable

AlexMenor committed:
a share link is unlimited when its timesValid value is -1

AlexMenor committed:
update sdk

AlexMenor committed:
add confirm delete dialog in shared links view

AlexMenor committed:
extract link settings into its own modal in SharedLinksView

AlexMenor committed:
avoid cropped menu in list component

AlexMenor committed:
add some shortcuts

AlexMenor committed:
prevent item list from being unselected if user deletes or opens item menu

AlexMenor committed:
upgrade sdk

AlexMenor committed:
add documentation for the generic list component

vicblab committed:
Removed the limited views feature

PixoDev commented on src/app/core/factory/sdk/index.ts:
```diff
@@ -47,9 +47,9 @@ export class SdkFactory {
+    const apiUrl = this.getApiV2Url();
     const appDetails = SdkFactory.getAppDetails();
-    const apiSecurity = this.getApiSecurity();
+    const apiSecurity = this.getNewApiSecurity();
```
whats the difference between the old and the new one?

vicblab committed:
First version of shared icon, Phosphor icons added

vicblab committed:
Unnecessary method removed

vicblab committed:
Icons' sizes increased

foxkdev committed:
Merge remote-tracking branch 'origin/shared-links-and-trash' into shared-links-and-trash

vicblab committed:
Commented provisional solution for shared items

foxkdev committed:
Merge remote-tracking branch 'origin/shared-links-and-trash' into shared-links-and-trash

foxkdev committed:
fixed code and timesValid

foxkdev committed:
fixed times valid to -1

foxkdev committed:
added code

sg-gs committed:
env: add REACT_APP_DRIVE_NEW_API_URL

sg-gs committed:
deps: upgrade @internxt/sdk to 1.4.0

sg-gs committed:
refactor(share): buildLinkFromShare method.

- Modified createShare, createShareLink, deleteShareLink, getSharedFileInfo functions

sg-gs committed:
fix(share): use fileId field instead of id

sg-gs committed:
refactor(share): use item field instead of fileMeta on shared file download

sg-gs committed:
refactor(share): use token field instead of itemToken on shared folder download

sg-gs committed:
refactor(share): adapt share generation to code usage (share version 2)

larryrider committed:
Fixed problems and inconsistencies on Sdk

larryrider committed:
removed '/api' from newApiInstance url

larryrider committed:
fixed checkbox warnings

larryrider committed:
hidden share link settings

larryrider committed:
block copy link when loading

larryrider committed:
fixed bug when last link(s) are removed

larryrider committed:
fixed delete shared links dialog

larryrider committed:
fixed not being able to download again on shared link page

larryrider committed:
fixed file preview

larryrider committed:
added fileSize to sharedLinks

larryrider committed:
fixed no scroll list bug

larryrider committed:
fixed download shared folder

larryrider committed:
added increment shared link view logic

larryrider committed:
replaced incrementShareView id with token

larryrider committed:
replaced incrementShareView id with token

larryrider committed:
updated package.json

larryrider committed:
removed empty items from shared links

larryrider committed:
updated sdk on yarn.lock

larryrider committed:
Merge branch 'master' into shared-links-and-trash

larryrider committed:
removed Trash from menu

larryrider committed:
Merge branch 'master' into shared-links-and-trash
